solution_mosaic.cpp

```diff
@@ -34,34 +34,39 @@ void drawKeypointMatches(cv::Mat& vis_img,
 void drawEstimationDetails(cv::Mat& vis_img, DurationInMs est_duration, size_t num_inliers);
 
 
-void lab4()
+void runSolutionMosaic()
 {
   // Open video stream from camera.
   const int camera_id = 0; // Should be 0 or 1 on the lab PCs.
   cv::VideoCapture cap(camera_id);
+
+  // Set frame size.
+  const int frame_cols = 640;
+  const int frame_rows = 480;
```

Comment:
Når du først har `const`, så kunne du vel tatt `constexpr` på disse :smirk:

Reply:
Ut fra resten av koden så lurer jeg på om vi valgte å droppe `constexpr` for leselighet :thinking: